packages/at_secondary_server/pubspec.yaml

```diff
@@ -26,13 +26,12 @@ dependencies:
   at_persistence_spec: 2.0.12
   at_persistence_secondary_server: 3.0.52
   version: 3.0.2
-  meta: 1.8.0
+  meta: ^1.9.1
```

Comment:
Rather than `^1.9.1` can we not go with `1.9.1` in line with the other dependencies here?

Reply:
Sure Chris, Removed the caret symbol before the version.